src/test/resources/services.conf

```diff
@@ -19,7 +30,7 @@
 
 
 services {
-  example-frontend{
+  economic-crime-levy-status-frontend{
```

Comment:
is this meant to be `returns` or `registration`? what is `status`?

Reply:
added return and registration services now

Reply:
pls keep `frontend` appended to the end, as we have backend services which we shouldn't get them mixed up with